spec/models/reports/repository_spec.rb

```diff
@@ -363,6 +373,18 @@ def refresh_views
   end
 
   context "legacy control specs" do
+    it "works for very old dates" do
+      facility_1 = create(:facility)
+      patient = create(:patient, registration_facility: facility_1, recorded_at: jan_2020.advance(months: -4))
+      create(:blood_pressure, :under_control, recorded_at: jan_2020.advance(months: -1), patient: patient, facility: facility_1)
+      refresh_views
+
+      ten_years_ago = patient.recorded_at.advance(years: -10).to_period
+      range = ten_years_ago..(ten_years_ago.advance(months: 12))
+      repo = Reports::Repository.new(facility_1, periods: range)
+      expect(repo.adjusted_patient_counts[facility_1.slug]).to eq({})
```

Comment:
I'm not sure I understand why we expect this to be empty. Is it because the patient is no longer hypertensive or is now LTFU? The test does validate that it doesn't bomb with older facilities, but it doesn't really test much function.

Reply:
The result set would be. empty because the range we are selecting for (10 years ago) in the Repository is before the registration date of the patient.